src/design/optForceGAMS/testOptForceSol.m

```diff
@@ -0,0 +1,116 @@
+function [maxGrowthRate, minTarget, maxTarget] = testOptForceSol(model, targetRxn, solution, relax, tol)
+%% DESCRIPTION
```

Comment:
Sorry for the misunderstanding. testOptForceSol is not a test function. This function analize if the solution found by optForce guarantee a minimum production of the target metabolite. I think I will change the name of the function to avoid misunderstandings